front/lib/resources/subscription_resource.ts

```diff
@@ -34,6 +45,90 @@ export class SubscriptionResource extends BaseResource<Subscription> {
     return new SubscriptionResource(Subscription, subscription.get());
   }
 
+  static async fetchByWorkspace(
+    workspace: LightWorkspaceType
+  ): Promise<SubscriptionType> {
+    const res = await SubscriptionResource.fetchByWorkspaces([workspace]);
+
+    const subscription = res[workspace.sId];
+    if (!subscription) {
+      throw new Error(
+        `Could not find subscription for workspace ${workspace.sId}`
+      );
+    }
+
+    return subscription;
+  }
+
+  static async fetchByWorkspaces(
```

Comment:
`nit` Given that it fetches only active subscription.
```suggestion
  static async fetchActiveByWorkspaces(
```